README.md

```diff
@@ -237,19 +236,19 @@ to the particular application of the transparent store; this is known as a
 The primary purpose of a personality is to implement **admission criteria** for
 the store, so that only particular types of data are added to the store. For
 example, a certificate transparency log only accepts data items that are valid
-certificates; a "CT Log" personality would police this, so that the Trillian
+certificates; a "CT Log" personality would police this so that the Trillian
 service can process all incoming data blindly.
 
-A personality may also perform **canonicalization** on incoming data, to
+A person may also perform **canonicalization** on incoming data, to
```

Comment:
Definitely "personality" - this is what we call the applications which build on and use Trillian.